sorbet/config

```diff
@@ -0,0 +1,4 @@
+--dir
+.
+--ignore=tmp/
+--ignore=vendor/
```

Comment:
I'm not sure about these configurations. There is no `tmp/` and `vendor/` in my setup.

Reply:
We use `BUNDLE_PATH: "vendor/bundle"` in the installer. `tmp` is redundant there.